re.sonny.Commit.json

```diff
@@ -22,6 +22,39 @@
     "*.a"
   ],
   "modules": [
+    {
+        "name" : "libsass",
+        "buildsystem" : "meson",
+        "sources" : [
+            {
+                "type" : "git",
+                "url" : "https://github.com/lazka/libsass.git",
+                "branch" : "meson"
+            }
+        ]
+    },
+    {
+        "name" : "sassc",
+        "buildsystem" : "meson",
+        "sources" : [
+            {
+                "type" : "git",
+                "url" : "https://github.com/lazka/sassc.git",
+                "branch" : "meson"
+            }
+        ]
+    },
+    {
+        "name" : "libadwaita",
+        "buildsystem" : "meson",
+        "sources" : [
+            {
+                "type" : "git",
+                "url" : "https://gitlab.gnome.org/GNOME/libadwaita.git",
+                "branch" : "main"
+            }
+        ]
+    },
```

Comment:
Doesn't GNOME 41 include libadwaita?

Any specific reason not to use fixed versions?

I guess gspell module should be removed.

Reply:
* GNOME 41 doesn't include libadwaita
* In-tree development manifests typically don't pin upstream
* Will remove gspell

Reply:
> In-tree development manifests typically don't pin upstream

Feel free to leave as is - I may pin them later. I hate bad surprises :smile:

> Will remove gspell

Thanks